src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -3171,4 +3171,7 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Symbol_search" xml:space="preserve">
     <value>Symbol search</value>
   </data>
+  <data name="required_property" xml:space="preserve">
+    <value>required property</value>
```

Comment:
Since "required property" is a C#-only concept, consider moving this to `CSharpFeaturesResources` as well.

I apologize for the fact, that I'm producing only a few comments at a time instead of leaving all of them at once. I am not an experienced reviewer, so my eye catches things gradually over time

Reply:
Good point, moved to `CSharpFeaturesResources`